gnomad_constraint/pipeline/constraint_pipeline.py

```diff
@@ -175,6 +177,36 @@ def main(args):
                 ).write(training_resources[region].path, overwrite=overwrite)
             logger.info("Done with creating training dataset.")
 
+        # Build plateau and coverage models for autosomes/pseudoautosomal regions,
+        # chromosome X, and chromosome Y
+        if args.build_models:
+            plateau_models = {}
+            # Check if the training datasets exist.
+            check_resource_existence(
+                input_pipeline_step="--create-training-set",
+                input_resources=training_resources.values(),
+                overwrite=overwrite,
+            )
+            # Build plateau and coverage models.
+            for region in GENOMIC_REGIONS:
+                if region == "autosome_par":
+                    logger.info("Building %s plateau and coverage models...", region)
+                    coverage_model, plateau_models[region] = build_models(
+                        training_resources[region].ht(),
+                        weighted=use_weights,
+                        pops=POPS if use_pops else (),
+                        build_coverage_model=True,
+                    )
+                    logger.info("Done building %s plateau and coverage models.", region)
+                else:
+                    logger.info("Building %s plateau models...", region)
+                    _, plateau_models[region] = build_models(
+                        training_resources[region].ht(),
+                        weighted=use_weights,
+                        pops=POPS if use_pops else (),
+                    )
+                    logger.info("Done building %s plateau models.", region)
```

Comment:
the models get built but not written, right? should they get stored as a pickle file or in an ht's global fields so this code doesn't need to be run more than once?